assembl/static2/js/app/components/voteSession/gaugeVoteForProposal.jsx

```diff
@@ -0,0 +1,292 @@
+// @flow
+/* eslint-disable react/no-multi-comp */
+import React from 'react';
+import Slider from 'rc-slider';
+import Pointer from '../svg/pointer';
+
+type Choice = {
+  id: string,
+  label: string,
+  value: number
+};
+
+type GaugeVoteForProposalProps = {
+  id: string, // the vote specification id
+  proposalId: string,
+  voteForProposal: Function,
+  instructions: string,
+  choices: ?Array<Choice>,
+  value: number
+};
+
+type GaugeVoteForProposalState = {
+  value: number
+};
+
+type NumberGaugeVoteForProposalState = {
+  value: number
+};
+
+const gaugeHeight = '12px';
+
+const markStyle = {
+  marginTop: '7px'
+};
+
+const trackStyle = [
+  {
+    visibility: 'visible',
+    height: gaugeHeight
+  }
+];
+
+const railStyle = {
+  backgroundColor: '#E6E5F4',
+  height: gaugeHeight
+};
+
+/* dotStyle prop of rc-slider mysteriously does not work anymore, so we declare these rules in the SCSS file
+const dotStyle = {
+  backgroundColor: '#E6E5F4', // TODO: use theme colors
+  height: '20px',
+  top: '0',
+  width: '2px',
+  border: 'none',
+  borderRadius: 'initial',
+  marginLeft: '0'
+};
+*/
+
+const handleStyle = [
+  {
+    height: '0',
+    width: '0',
+    marginTop: '-20px',
+    border: 'none',
+    boxShadow: 'none'
+  }
+];
+
+const Handle = Slider.Handle;
+const handleIcon = (props) => {
+  const { value, dragging, ...restProps } = props;
+  const style = {
+    marginTop: '-15px',
+    cursor: '-webkit-grab'
```

Comment:
Do we need this proprietary cursor type? What does it show in the different browsers?

Reply:
@swergas what do you think?